web/vtubers.ts

```diff
@@ -92,6 +94,20 @@ export const vtubers: Record<VTuberIds, VTuber> = {
     default: true,
     image: "e36068cc-9bfb-4f9b-dcbe-549e81f7db00",
   },
+  hololive_en: {
+    id: "hololive_en",
+    twitter: "hololive_En",
+    youtube: "UCotXwY6s8pWmuWd_snKYjhg",
+    default: true,
+    image: "e36068cc-9bfb-4f9b-dcbe-549e81f7db00",
```

Comment:
no idea what value to use for image properties

Reply:
It stands for cloudflare images id.